scr/components/Key.jsx

```diff
@@ -0,0 +1,72 @@
+import { useCallback, useState, useEffect, useMemo } from "react";
+import { Api } from "../utils/Api";
+import { Button } from "./Button";
+
+import cn from "classnames";
+
+export const Key = ({
+  value,
+  removeKey,
+  incrementUsedKeys,
+  decrementUsedKeys,
+}) => {
+  const [isUsed, setUsedState] = useState(false);
+  const [isLoading, setLoadingState] = useState(false);
+
+  const toggleLoading = useCallback(() => {
+    setLoadingState(!isLoading);
```

Comment:
как и в App.jsx лучше убрать ненужную зависимость от isLoading
```suggestion
    setLoadingState(prevState => !prevState);
```